addon/wait-for.js

```diff
@@ -4,31 +4,25 @@ const {
   $,
   Test: {
     registerAsyncHelper,
-    _helpers: { find: { method: find } }
   },
   RSVP: { Promise }
 } = Ember;
 
-function _waitFor(
-  app,
-  selector,
-  context,
-  {
-    count = 1,
-    interval = 1
-  } = {}
-) {
-  let _find;
-  if (app) {
-    _find = find;
+function _waitFor(app, selectorOrFn, interval = 1) {
```

Comment:
Then you could include `interval` (deprecated), and feed it to `selectorToExist` below, then I don't think there would be a breaking change.